functions/equals-text/1.0/index.js

```diff
@@ -0,0 +1,41 @@
+const equalsText = ({ leftValue, rightValue, comparator, toLower }) => {
+  console.log("🚀 ~ file: index.js:2 ~ equalsText ~ comparator", comparator)
+  let result = false;
```

Comment:
Don't declare the variable. Better practice to return from switch and have the defaulted value be `false`.

Reply:
This is what I had first so I kindly refuse to go back again, not going to rebuild what was first denied.